CONTRIBUTING.md

```diff
@@ -30,6 +35,9 @@ requests; here are our guidelines:
 - Use the same coding conventions as the rest of the project.
 - Open a [pull request][6] that relates to *only* one subject with a clear title
   and description in grammatically correct, complete sentences.
+- If a PR contains a fix or improvement for a release branch, 2.x for instance, that PR should be raised
```

Comment:
we should either explain what a release branch here is or just remove the word. This is adding confusion.

Reply:
It's a standard term in software engineerin. Please see [1] for an explanation.

[1] https://producingoss.com/en/release-branches.html